templates/terraform/resource.erb

```diff
@@ -217,21 +217,24 @@ func resource<%= resource_name -%>Create(d *schema.ResourceData, meta interface{
 
 <%  if !object.async.nil? && object.async.allow?('create') -%>
 <%    if object.async.is_a? Api::PollAsync -%>
-    waitURL, err := replaceVars(d, config, "<%= "{{#{object.__product.name}BasePath}}#{object.self_link_uri}" -%>")
+    pollUrl, err := replaceVars(d, config, "<%= "{{#{object.__product.name}BasePath}}#{object.self_link_uri}" -%>")
```

Comment:
nit: go style is URL (https://github.com/golang/go/wiki/CodeReviewComments#initialisms)

Reply:
herp